process/smartContract/process.go

```diff
@@ -1250,6 +1250,22 @@ func (sc *scProcessor) addBackTxValues(
 		scrIfError.Value = big.NewInt(0).Set(valueForSnd)
 	}
 
+	if sc.flagSenderInOutTransfer.IsSet() &&
```

Comment:
this will activate in the imminent next release. If can not merge this PR yet as the master is frozen. Please use another flag.

Reply:
the idea is to merge this to master right now